browser/ui/views/brave_actions/brave_shields_action_view.cc

```diff
@@ -189,7 +189,7 @@ void BraveShieldsActionView::ButtonPressed() {
   if (!webui_bubble_manager_) {
     webui_bubble_manager_ =
         std::make_unique<WebUIBubbleManagerT<ShieldsPanelUI>>(
-            this, profile_, GURL(kShieldsPanelURL), 1);
+            this, profile_, GURL(kShieldsPanelURL), IDS_BRAVE_SHIELDS);
```

Comment:
I'm thinking we should use "Brave Shields UI" so that people don't get confused and think this is a process for the actual shields work, and know this is for the UI renderer.

Reply:
the name has "Tool: ..." as part of the string

Reply:
can do it as a follow up